sys-libs/pam/pam-1.5.1.ebuild

```diff
@@ -33,6 +33,7 @@ DEPEND="
 	berkdb? ( >=sys-libs/db-4.8.30-r1:=[${MULTILIB_USEDEP}] )
 	selinux? ( >=sys-libs/libselinux-2.2.2-r4[${MULTILIB_USEDEP}] )
 	nis? ( net-libs/libnsl[${MULTILIB_USEDEP}]
+	xcrypt? ( >=sys-libs/glibc-2.30[-crypt] sys-libs/libxcrypt[system] )
```

Comment:
I think instead depend on >=virtual/libcrypt-2:= which does this, but doesn't break on non-glibc.